focus-ios/focus-ios-tests/XCUITest/TPSettingsTest.swift

```diff
@@ -4,7 +4,8 @@
 
 import XCTest
 
-class TrackingProtectionSettings: BaseTestCase {
+class TrackingProtectionTest: BaseTestCase {
```

Comment:
The test suite has the new name but not the Test file, it is still named TPSettingsTest.swift, it would be nice to have the same name